tests/integration/update.bats

```diff
@@ -76,6 +78,10 @@ function check_cgroup_value() {
     [ "$status" -eq 0 ]
     check_cgroup_value $CGROUP_BLKIO "blkio.weight" 500
 
+    # update blkio-leaf-weight
+    runc update test_update --blkio-leaf-weight 500
+    check_cgroup_value $CGROUP_BLKIO "blkio.leaf_weight" 500
```

Comment:
@cyphar  thanks for checking this PR. For other parameters of blkio, we cannot test them in current CI machine, because it is lack of cfq device in CI machine. What is your suggestion about the device limitation of CI machine?